app/elements/profile/profile-page.html

```diff
@@ -0,0 +1,138 @@
+<link rel="import" href="/bower_components/iron-image/iron-image.html">
+<link rel="import" href="/bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="/bower_components/paper-material/paper-material.html">
+
+<dom-module id="profile-page">
+
+	<style>
+		:host{
+			display: block;
+			height: 100%;
+		}
+		paper-material{
+			background: white;
+		}
+		#topLeft{
+			overflow: auto;
+			margin: 16px 8px 8px 16px;
+		}
+		#topRight{
+			margin: 16px 16px 8px 8px;
+		}
+		#midLeft{
+			margin: 8px 8px 8px 16px;
+		}
+		#midRight{
+			margin: 8px 16px 8px 8px;
+		}
+		.card{
+			height: calc(100% - 56px);
+			width: calc(50% - 48px);
+			padding: 16px;
+		}
+		iron-image{
+			width: 300px;
+			height:300px;
+		}
+		#userInfo{
+			margin-left: 16px;
+			word-wrap: break-word;
+		}
+	</style>
+
+	<template>
+		<iron-ajax
+			id="getUserRequest"
+			handle-as="json"
+			on-response="onGetUserResponse"
+			on-error="onGetUserError">
+		</iron-ajax>
+		<iron-ajax
+			id="getProfileRequest"
+			handle-as="json"
+			on-response="onGetProfileResponse">
+		</iron-ajax>
+		<iron-ajax
+			id="getLatestChallenges"
+			handle-as="json"
+			on-response="onLastChallengesResponse"
+			on-error="onLastChallengesError">
+		</iron-ajax>
+		<div class="vertical layout" class="fullHeight">
+			<div class="horizontal layout" class="halfHeight">
+				<paper-material id="topLeft" elevation="1" class="card">
+					<div class="horizontal layout">
+						<iron-image sizing="content" src="http://lorempixel.com/400/400"></iron-image>
```

Comment:
there's still a link to Lorem Pixel here. Can't we fall back to the Letter as we do in the list view?